modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1365,6 +1368,14 @@ components:
           scopes:
             'write:pets': modify pets in your account
             'read:pets': read your pets
+    global_api_key_header:
+      type: apiKey
+      name: global_api_key_header
+      in: header
+    global_api_key_cookie:
+      type: apiKey
+      name: global_api_key_cookie
+      in: cookie
```

Comment:
can you please copy this file to  `modules/openapi-generator/src/test/resources/3_0/k6/petstore-with-fake-endpoints-models-for-testing.yaml` and revert the change in this file which is used by many other generators for testing?

please also update https://github.com/OpenAPITools/openapi-generator/blob/master/bin/configs/k6.yaml#L3 with the new path to the test spec used by k6

Reply:
thanks. looks good

if there's no further feedback from @mostafa, i will merge it early next week

have a nice weekend